File: Python/car.py
from account import Account

class Car:
    id = int
    license = str
    driver = Account
    passenger = str

    def __init__(self, license, driver):
        self.license = license
        self.driver = driver


    def print_data_car(self):
        print("License: {}, Name: {}, Documet: {}".format(self.license, self.driver.name, self.driver.document))

File: JavaScript/Car.js
class Car {
  constructor(license, driver) {
    this.id;
    this.license = license;
    this.driver = driver;
    this.passenger;
  }

  print_data_car() {
    console.log(this.driver);
    console.log(this.driver.name);
    console.log(this.driver.document);
    console.log(this.license);
  }
}
